GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.ListObjects.cs

```diff
@@ -26,13 +26,15 @@ public partial class StorageClient
         /// Asynchronously lists the objects in a given bucket, returning the results as a list.
         /// </summary>
         /// <remarks>
-        /// This is a convenience method for calling <see cref="ListAllObjectsAsync(string, ListObjectsOptions, CancellationToken)"/>.
+        /// This is a convenience method for calling <see cref="ListAllObjectsAsync(string, string, ListObjectsOptions, CancellationToken)"/>.
         /// </remarks>
         /// <param name="bucket">The bucket to list the objects from. Must not be null.</param>
+        /// <param name="prefix">The prefix to match. Only buckets with names that start with this string will be returned.
```

Comment:
I think you may want to refer to _objects_ instead of _buckets_ as in _Only objects with names that start with..._.